apph-back/src/main/java/com/viseo/apph/dao/PhotoDao.java

```diff
@@ -27,4 +27,9 @@ public List<Photo> getUserPhotos(User user) {
     public Photo getPhoto(long id) {
         return em.find(Photo.class, id);
     }
+
+    public void deletePhoto(long id) {
+        Photo photo = em.find(Photo.class, id);
+        em.remove(photo);
```

Comment:
Si tu as déjà récupérer la photo dans le service tu peux directement mettre en paramètre la photo plutôt que l'id.
```suggestion
    public void deletePhoto(Photo photo) {
        em.remove(photo);
```